Makefile

```diff
@@ -241,13 +241,16 @@ all: build update-workflows
 
 build: $(BUILD_BINS)
 
+build-static: clean-bin
```

Comment:
Should we add the `clean-bin` dep to `build:` instead? Just thought about altering static and non-static builds...

Reply:
Hmm, yes. Maybe that's a better idea.

Reply:
Okay, I updated it so, that binaries are rebuilt if
1. they are out of date, or
2. existing binaries were built with a different STATIC setting than the current one